### DISCUSSION TITLE
reportCrash shows up in Sentry

### DISCUSSION THREAD

**Discussion by janwiebe-jump** [infinitered/ignite#2648] [Q&A]
When calling reportCrash from within my code, to log the error to Sentry, it always shows up as 'reportError'.
The actual error is somewhat hidden in the subtitle.
Any ideas to make this a more useful display? For example the source where the reportCrash was called?

![image](https://github.com/infinitered/ignite/assets/64576907/5d173336-8481-4938-bf78-1b299690f4a9)

**Comment by frankcalise** [comment]
Hi @janwiebe-jump. I moved this to a discussion as it's not directly an issue related with Ignite but you seem to have some questions about best practices around Sentry.

Could you give us some more info so we can help you? What version of Ignite/Expo/Sentry are you using. Perhaps the package json and where your try/catch is where you are reporting the error?

You can also enhance the reporting feature to include level, etc. See the docs here: https://docs.sentry.io/platforms/react-native/usage/set-level/

**Comment by janwiebe-jump** [comment]
I changed the issue grouping in my Sentry project settings, by adding a line for the `crashReporting.ts`file, to exclude it from grouping like this.

![image](https://github.com/infinitered/ignite/assets/64576907/07ea241d-e1c5-4112-bfd1-63cc0055d027)

Based on [this documentation](https://docs.sentry.io/product/data-management-settings/event-grouping/stack-trace-rules/)
